src/test/crimson/seastore/test_seastore.cc

```diff
@@ -518,3 +548,102 @@ TEST_F(seastore_test_t, simple_extent_test)
     test_obj.check_size(*seastore);
   });
 }
+
+TEST_F(seastore_test_t, fiemap_empty)
+{
+  run_async([this] {
+    auto &test_obj = get_object(make_oid(0));
+    test_obj.touch(*seastore);
+    test_obj.truncate(*seastore, 100000);
+
+    std::map<uint64_t, uint64_t> m, e;
+    m = test_obj.fiemap(*seastore, 0, 100000);
+    cout << " got " << m << std::endl;
```

Comment:
no news is good news. i'd avoid flooding the stdout if the test succeeds.

Reply:
done.